features/basics/spies.feature

```diff
@@ -125,5 +125,5 @@ Feature: Spies
       |           received: 2 times with any arguments                                                   |
       |                                                                                                  |
       |  2) An invitiation fails when an order constraint is not satisifed                               |
-      |     Failure/Error: expect(invitation).to have_received(:deliver).with("foo@example.com").ordered |
+      |     Failure/Error: expect(invitation).to have_received(:deliver).with("bar@example.com").ordered |
```

Comment:
I thought the first expectation (with bar) was the one that should be throwing the error, but I realize now that the first expectation should pass since it's a spy. It's the second expectation that should actually fail. I'll need to adjust the pull request a bit to adjust for this.